lexos/static/js/scripts_viz.js

```diff
@@ -1,5 +1,11 @@
+import d3 from './Lexos/lexos/static/node_modules/d3'
```

Comment:
I'm not sure if this `import` statement would work. Could you please double check?